### ISSUE TITLE
Kingdom Hearts HD 1.5 ReMIX [BLUS31212]

### ISSUE THREAD

**Issue by Baxusfox** [RPCS3/rpcs3#4851]
- RPCS3 0.0.5-7044-6a62de6d6 Alpha_Win64

- After finishing the Kingdom Hearts 1 game and watching the last cutscene after the credits, the game freezes in a black screen and does not show the ending stats for the completion of the game thus does not save after beating the boss.

- Intel(R) Core(TM) i7-3770 @ 3.40GHz
- 16 GB DDR3
- NVIDIA GeForce GTX 1070

- [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/2168892/RPCS3.log.gz)
- [Video Log](https://youtu.be/4-V0xFILLqM)

**Comment by LorenzoRubagotti** [comment]
Hi, can you share your configuration file? Do you know if the mermaid world work fine?

**Comment by Baxusfox** [comment]
[config.zip](https://github.com/RPCS3/rpcs3/files/2172229/config.zip)

Atlantica is still having [#4303](https://github.com/RPCS3/rpcs3/issues/4303). Although, you can use this [PR](http://www.mediafire.com/file/hyg2v1iqvvjh6sc/Rpcs3.zip/file) as a work around. Just copy your dev_hdd0 into the folder.

**Comment by LorenzoRubagotti** [comment]
Thank you very much. Do you know if i have to copy all the dev_hdd0 or only the 00000001 in dev_hdd0\home?

I think that this issue there is also in the intro video. Sometime the game freezes after the intro video(when you wait for a bit of time in the menu).

I can confirm that the problem is not caused by Windows 10 insider preview release because i have not insider preview windows 10  and I have this issue me too

**Comment by Baxusfox** [comment]
Just 00000001 is fine.

**Comment by Baxusfox** [comment]
Yes, 00000001 in home is fine. I'm also getting the same issue after the intro video.

**Comment by LorenzoRubagotti** [comment]
Perhaps they are due to the same bug

**Comment by Baxusfox** [comment]
I hope so.

**Comment by kyleondar** [comment]
This bug does not only exist for KH 1, but for ReCoM as well, I just finished the final boss 3x in a row hopping I would be able to cut scene skip it to reach a save point. But all 3 times it crashed on me. So it is affecting both games on the disk

**Comment by hufman** [comment]
In general, it seems that cutscene videos are unstable. The opening cutscene sometimes freezes after the video plays, with the workaround being to move the OPN.pam file out of the way. I just experienced a freeze when waiting at the start menu long enough for the trailer video to start, and then pushing a button freezes at a white screen, 50% of the time in 0.0.5-7405-da6ce80f.

Suspicious log entries include:
```
E {PPU[0x100000a] Thread (KH1 dkThread) [0x000448d8]} Stat: 'sys_semaphore_wait' failed with 0x80010005 : CELL_ESRCH [x296]
E {PPU[0x100000a] Thread (KH1 dkThread) [0x000448d8]} Stat: 'sys_semaphore_post' failed with 0x80010005 : CELL_ESRCH [x296]
E {PPU[0x1000028] Thread (_sys_MixerChStripMain) [0x00269a28]} Stat: '_sys_lwmutex_lock' failed with 0x80010005 : CELL_ESRCH [x75328]
E {PPU[0x100002a] Thread (_cellsurMixerMain) [0x00265f00]} Stat: 'sys_mutex_lock' failed with 0x80010008 : CELL_EDEADLK [x9416]
E {PPU[0x1000000] Thread (main_thread)} Stat: 'sys_semaphore_post' failed with 0x80010005 : CELL_ESRCH [x363]
E {PPU[0x1000000] Thread (main_thread)} Stat: '_sys_lwmutex_lock' failed with 0x80010005 : CELL_ESRCH [x6]
E {PPU[0x1000000] Thread (main_thread)} Stat: 'sys_semaphore_wait' failed with 0x80010005 : CELL_ESRCH [x363]
```

**Comment by kyleondar** [comment]
But as I was stating I finished the game, i did the cut scene skip youre referring too at every point in the game to allow me to progress. But because it is the final boss, there is no chance to skip the cut scene so youre stuck in an infinite loop of win the game and it locks up. restart, win the game, locks up. ect ect in the hopes that 1 time it will let the game finish so that you may start on the riku play thru

**Comment by elad335** [comment]
Fixed by #5310 ?

**Comment by bobdachef** [comment]
> But as I was stating I finished the game, i did the cut scene skip youre referring too at every point in the game to allow me to progress. But because it is the final boss, there is no chance to skip the cut scene so youre stuck in an infinite loop of win the game and it locks up. restart, win the game, locks up. ect ect in the hopes that 1 time it will let the game finish so that you may start on the riku play thru

I have the same issue. Would importing a Riku save file work?

**Comment by Fizzwin** [comment]
Is it already fixed?

**Comment by LindsNemesis** [comment]
v0.0.6-8000 Alpha [2019-04-19]

Have the same issue but in **ANY** Kingdom Hearts, freeze occurs in Menu Cutscene too 

Microsoft Windows 10 Pro
Intel(R) Core(TM) i7-4790K @ 4.4GHz
16 GB DDR3
NVIDIA GeForce GTX 1080TI

Video: [Menu Freeze](https://www.youtube.com/watch?v=dbiTlf2HF3U&feature=youtu.be)

[RPCS3 (1).log.gz](https://github.com/RPCS3/rpcs3/files/3099801/RPCS3.1.log.gz)

**Comment by darkash** [comment]
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/4110791/RPCS3.log.gz)
recent version log, upon reading the log, it seems like the freeze happen when it infinitely loading a certain spu over and over again, the log also grows so big thanks to this spam

**Comment by DiogoLuiz2** [comment]
It seems it will never be fixed.

**Comment by nitroos-ai** [comment]
May be #9655 comment could help.

**Comment by darkash** [comment]
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/7457805/RPCS3.log.gz)

updating the issue with new log, the problem still happens on 0.0.19 although it probably have different message compred to last log I uploaded

> May be @elad335 comment could help in #9655.

doing this in multiple threads is the fastest way to get you blocked, if you wanna ask a dev, do mention in the issue instead of mentioning in an unrelated issue threads

**Comment by Darkhost1999** [comment]
Here is a save file to help testing
[BLUS31212-BISLPS25198.zip](https://github.com/user-attachments/files/23443010/BLUS31212-BISLPS25198.zip)

**Comment by Asinin3** [comment]
Just going to mention in KH Chain of Memories had an issue with the intro cutscene that plays on title screen. It would freeze at the last frame before going back to title screen but it's fixed by libvdec.sprx. I also tested 358/2 Days and left the ~~movie~~game running for two hours with default settings and i had no issues without libvdec.sprx turned on.

Someone needs to retest the issue on credits with libvdec if they haven't already, i didn't see it in people's logs.

**Comment by Darkhost1999** [comment]
Got the trophy and everything although it never popped up or play sound.
https://user-images.githubusercontent.com/60384196/169700977-d43eca1a-5279-41de-9e06-a676f8c9a546.mp4

Fixed by LLE. So still a valid issue. What did we accomplish? A workaround?

**Comment by YuriNator557** [comment]
We have Playable games where libvdec.sprx is required, could that not be considered a fix?

**Comment by Megamouse** [comment]
no

**Comment by YuriNator557** [comment]
So should the games in question be moved back to Ingame then?

**Comment by Megamouse** [comment]
no

**Comment by Megamouse** [comment]
github isn't related to the compat list...
This is for development only

**Comment by digant73** [comment]
@Baxusfox, @Darkhost1999 is that still an issue?

**Comment by Darkhost1999** [comment]
[RPCS3.log](https://github.com/user-attachments/files/23859500/RPCS3.log)
Yes this is still an issue. You get the end credits. You see Pluto in the 100 acre woods, then You see the  message Remember Sora you are the one who will open the door. Followed by an eternal darkness of an infinite black screen. Tons of savedata errors All before the battle report.
All of this ending works fine when libvdec is LLE'd.

**Comment by Darkhost1999** [comment]
My comment earlier was also updated with a Sora save that has been leveled up slightly more than before. Makes for a relatively easy final boss fight with some new upgraded spells. Just don't forget to equip Donald with throw away potions, goofy with team help potions, and yourself with whatever you want. They shouldn't be needed unless you lean heavily on magic.
